openshift-cert-manager-operator/operator/base/kustomization.yaml

```diff
@@ -1,9 +1,6 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: cert-manager-operator
```

Comment:
Generally I prefer to keep the namespace in the kustomize, especially when you are dealing with multiple objects.

Reply:
I think most operators in the catalog use this pattern. Can we do this as a separate PR?